### ISSUE TITLE
Predicting from generator throws error

### ISSUE THREAD

**Issue by israelpf** [ilvkai/view-synthesis-for-vehicles#4]
When I do the generator.predict(batch) step the following error appears:

**tensorflow.python.framework.errors_impl.InvalidArgumentError: input_1:0 is both fed and fetched**

I am using Tensorflow 1.14 Keras 3.2.1

I would appreciate any advice with this issue.

Kind Regards.

**Comment by ilvkai** [comment]
Hi!
I suggest you use tensorflow-gpu==1.6.0 with Keras==2.2.0
Using other versions of tensorflow and keras may cause code and version incompatibility.
I hope this will help.

**Comment by israelpf** [comment]
Hi Sr.

Yes you are right, I have created an environment with the same specifications and it works, the main problem is that Tensorflow 1.6 and Keras 2.2 are out of support and they are missed in conda and pip repositories, so it is hard to get them now a days.

As a workaround I've cut the generator and I've created a new build of the model (i.e. Model(inputs=generator.inputs, outputs=output_layer.output) with this there is no loop for feeding the input layers in the model in newer versions of Keras-Tensorflow.

Thank you for you reply I appreciate it.

Regards.
